app/gui/view/component-browser/component-group/src/icon.rs

```diff
@@ -555,17 +555,17 @@ crate::define_icons! {
         ensogl::define_shape_system! {
             above = [crate::background, ensogl_list_view::background, ensogl_list_view::selection];
             (style: Style, strong_color: Vector4, weak_color: Vector4) {
-                let blue = style.get_color(theme::data_science::blue);
-                let rect1 = Rect((4.0.px(),4.0.px())).translate(((-5.5).px(),3.0.px())).fill(blue);
-                let rect2 = Rect((4.0.px(),4.0.px())).translate_y((-5.5).px()).fill(blue);
-
-                let gray = style.get_color(theme::data_science::gray);
-                let circle1 = Circle(2.0.px()).translate_y(5.5.px()).fill(gray);
-                let circle2 = Circle(2.0.px()).translate(((-5.5).px(),(-3.0).px())).fill(gray);
-                let circle3 = Circle(2.0.px()).translate((5.5.px(),(-3.0).px())).fill(gray);
+                let color = weak_color;
+                let circle = Circle(2.0.px());
+                let circle1 = circle.translate_y(5.5.px()).fill(color.clone());
+                let circle2 = circle.translate(((-5.5).px(),(-3.0).px())).fill(color.clone());
+                let circle3 = circle.translate((5.5.px(),(-3.0).px())).fill(color);
```

Comment:
I would put just `weak_color` instead of aliasing color. The same for the strong_color below.